Utils/Dataflow/055_documents2TTL/documents2ttl.py

```diff
@@ -1,7 +1,10 @@
 #!/usr/bin/env python
 """
-Module document2ttl.py
-- input from step 015 JSON:
+Stage for converting JSON files(output of stage 015) into TTL files(input for
+stage 060).
+
+JSON file should have the following structure::
```

Comment:
"Input JSON file (hereafter referred to as 'initial JSON') should..."?

Reply:
There is a definition of 'initial JSON' below the example. Do you suggest it to be moved here?

Reply:
Done.